packages/components/src/range-control/stories/index.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * External dependencies
+ */
+import { number, text } from '@storybook/addon-knobs';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import RangeControl from '../';
+
+export default { title: 'RangeControl', component: RangeControl };
+
+const RangeControlWithState = ( { label, help, min, max, beforeIcon, afterIcon, allowReset, ...props } ) => {
+	const [ value, setValue ] = useState( 5 );
+
+	return (
+		<RangeControl
+			{ ...props }
+			help={ help }
```

Comment:
All the props passed to range control excluding value and onChange come from RangeControlWithState. Could we just pass { ...props }, value, and onChange without defining any of the other variables in RangeControlWithState?

Reply:
Yes, this has been updated, thank you for pointing it out, much cleaner this way.